spec/lib/whitehall_importer_spec.rb

```diff
@@ -1,6 +1,51 @@
 # frozen_string_literal: true
 
 RSpec.describe WhitehallImporter do
+  describe ".create_migration" do
+    let(:whitehall_host) { Plek.new.external_url_for("whitehall-admin") }
+    let(:whitehall_export_page_1) {
```

Comment:
Convention is to use do and end for a multiline block.

Reply:
This is no longer an issue in my latest changes.